### PR TITLE
Zia data team review

### PR DESCRIPTION
**Pull request by evanluc**
For zia for review.

### CODE REVIEW

File: modules/data_team_helper/ajax/GetCandidates.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<?php
+header("content-type:application/json");
```
Comment (mohadesz) [inline modules/data_team_helper/ajax/GetCandidates.php]: Run phpcs

File: modules/data_team_helper/ajax/GetInstruments.php [hunk 2]
```diff
@@ -11,12 +12,42 @@
...
+        $flattened_result[$counter] = $v;
+        $counter++;
+    }
+
+    print json_encode($flattened_result);
+    exit();
+}
+
+//else a specific visit is set and only return visit instruments 
+else{
```
Comment (mohadesz) [inline modules/data_team_helper/ajax/GetInstruments.php]: run phpcs

File: modules/data_team_helper/js/data_team_helper.js [hunk 3]
```diff
@@ -23,62 +23,68 @@ function changeVisitLabels()
     var visit_label_value = visit_label_dropdown.value;
```
Comment (mohadesz) [inline modules/data_team_helper/js/data_team_helper.js]: run through jslint

File: modules/data_team_helper/php/NDB_Form_data_team_helper.class.inc [hunk 4]
```diff
@@ -51,146 +53,128 @@ class NDB_Form_data_team_helper extends NDB_Form
...
-            $this->form->addElement('static', "status", "No matching criteria found.");
-            return;
-        } else {
-            $this->form->addElement('static', 'total', 'Total number of results:', sizeof($info));
-        }
+        $incompleteCandidates = $this->getIncompleteCandidates($instrument,$_REQUEST['visit_label'], $candID);
+        $this->tpl_data['Incomplete_candidates'] = json_encode($incompleteCandidates);
+        $this->tpl_data['Incomplete_candidates_length'] = count($incompleteCandidates);
+
+        $conflicts= $this->InstrumentInConflict($instrument, $_REQUEST['visit_label'], $candID);
```
Comment (mohadesz) [inline modules/data_team_helper/php/NDB_Form_data_team_helper.class.inc]: run phpcs

File: modules/data_team_helper/ajax/GetCandidates.php (hunk 1 above)
Comment (evanluc) [inline modules/data_team_helper/ajax/GetCandidates.php]: Done.

File: modules/data_team_helper/ajax/GetInstruments.php (hunk 2 above)
Comment (evanluc) [inline modules/data_team_helper/ajax/GetInstruments.php]: Done.

File: modules/data_team_helper/js/data_team_helper.js (hunk 3 above)
Comment (evanluc) [inline modules/data_team_helper/js/data_team_helper.js]: Done.

**Comment by jstirling91** [comment]
The Behavioural Feedback tab doesn't load. Get error ```BehaviouralFeedbackRow is not defined```

**Comment by evanluc** [comment]
@jstirling91 

An E was missing ! Good catch.

**Pull request closed without merging**